include/glow/Quantization/Base/Base.h

```diff
@@ -91,9 +91,10 @@ inline DestTy quantize(float input, const TensorQuantizationParams &TQP) {
 
 /// Converts a quantized value (type eTy) to floating point based on the
 /// quantization parameters \p TQP.
+/// Note: use int64_t to cover the 'symmetric int32 with unsigned' case.
```

Comment:
does int32 quantization make sense? it does not bring any memory savings and I'm not sure about perf considering extra operations needed for int32.

Reply:
I think int32 or uint32 will only be fore specific cases, like scalars.
For instance already have a case for int32 with the convolution bias.  
Note that int64_t is only needed here for handling uint32, not int32.